core/modules/widgets/select.js

```diff
@@ -51,7 +51,12 @@ SelectWidget.prototype.render = function(parent,nextSibling) {
 Handle a change event
 */
 SelectWidget.prototype.handleChangeEvent = function(event) {
-	var value = this.getSelectDomNode().value;
+	if(this.selectMultiple == false) {
+		var value = this.getSelectDomNode().value;
+	} else {
+		var value = this.getSelectValues()
```

Comment:
I'll update this however we'd like, as soon as there's a consensus.  @Jermolene, oh BDFL, will you bring clarity to our muddled musings? :-P